R/contact_matrix.r

```diff
@@ -419,13 +418,31 @@ contact_matrix <- function(survey, countries=c(), survey.pop, age.limits, filter
                         survey$participants[, median(get(columns[["year"]]), na.rm=TRUE)]
                 }
             }
+        } else{
+            # if survey.pop is a data frame with columns 'lower.age.limit' and 'population'
+            survey.pop <- data.table(survey.pop)
+            # make sure the maximum survey.pop age exceeds the participant age group breaks
+            if(max(survey.pop$lower.age.limit) < max(part.age.group.present)){
```

Comment:
Really minor point but I think it would be good to keep with the formatting of the rest of the code:
- space after `if`
- space before `{` or `(` (unless there's another opening bracket)
- space after `,` and around operators e.g. `+`

Reply:
Thank you for the formatting guidelines. I'll update the code once all code-logic issues have been resolved. Otherwise, merging different PR can become a nightmare (I suppose).

Reply:
As far as I'm concerned this is ready to be merged so I'd suggest to go ahead and update the formatting